html-forms/index.html

```diff
@@ -7,18 +7,18 @@
     <h1>Coding Form!</h1>
 
     <form>
-      <label for="mail">Email</label>
+      <label id="email">Email</label>
       <div>
-        <input type="email" name="user-email" id="mail" />
+        <input type="email" name="user-email" id="emailenter" />
       </div>
 
-      <label for="why-cool"> Why is coding cool?</label>
+      <label id="why-cool"> Why is coding cool?</label>
       <div>
         <select>
```

Comment:
The `id="why=cool"` needs to go on the `select`, but the `label` needs a `for` with the same value